client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapper.java

```diff
@@ -78,9 +75,46 @@ public abstract class OpenApiClientGeneratorWrapper {
         defaultTypeMappings.forEach(this.configurator::addTypeMapping);
         defaultImportMappings.forEach(this.configurator::addImportMapping);
 
+        this.setDefaults();
+
         this.generator = new DefaultGenerator();
     }
 
+    /**
+     * A few properties from the "with*" methods must be injected in the Qute context by default since we turned strict model
+     * rendering.
+     * This way we avoid side effects in the model such as "NOT_FOUND" strings printed everywhere.
+     *
+     * @see <a href="https://quarkus.io/guides/qute-reference#configuration-reference">Qute - Configuration Reference</a>
+     */
+    private void setDefaults() {
+        // Set default values directly here
+        this.configurator.addAdditionalProperty("additionalApiTypeAnnotations", new String[0]);
+        this.configurator.addAdditionalProperty("additionalPropertiesAsAttribute", FALSE);
+        this.configurator.addAdditionalProperty("additionalEnumTypeUnexpectedMember", FALSE);
+        this.configurator.addAdditionalProperty("additionalEnumTypeUnexpectedMemberName", "");
+        this.configurator.addAdditionalProperty("additionalEnumTypeUnexpectedMemberStringValue", "");
+        this.configurator.addAdditionalProperty("additionalRequestArgs", new String[0]);
+        this.configurator.addAdditionalProperty("classes-codegen", new HashMap<>());
+        this.configurator.addAdditionalProperty("circuit-breaker", new HashMap<>());
+        this.configurator.addAdditionalProperty("configKey", "");
+        this.configurator.addAdditionalProperty("datatypeWithEnum", "");
+        this.configurator.addAdditionalProperty("enable-security-generation", TRUE);
+        this.configurator.addAdditionalProperty("generate-part-filename", FALSE);
+        this.configurator.addAdditionalProperty("mutiny", FALSE);
+        this.configurator.addAdditionalProperty("mutiny-operation-ids", new HashMap<>());
+        this.configurator.addAdditionalProperty("mutiny-return-response", FALSE);
+        this.configurator.addAdditionalProperty("part-filename-value", "");
+        this.configurator.addAdditionalProperty("return-response", FALSE);
+        this.configurator.addAdditionalProperty("skipFormModel", TRUE);
+        this.configurator.addAdditionalProperty("templateDir", "");
+        this.configurator.addAdditionalProperty("use-bean-validation", FALSE);
+        this.configurator.addAdditionalProperty("use-field-name-in-part-filename", FALSE);
+        this.configurator.addAdditionalProperty("verbose", FALSE);
+        // TODO: expose as properties
+        this.configurator.addAdditionalProperty(CodegenConstants.SERIALIZABLE_MODEL, FALSE);
+    }
+
     public OpenApiClientGeneratorWrapper withApiPackage(final String pkg) {
```

Comment:
`withApiPackage` and `withModelPackage` seem no longer used. Can we remove them?